dataInterface/movies.js

```diff
@@ -76,32 +87,37 @@ module.exports.create = async (newObj) => {
   const database = client.db(databaseName);
   const movies = database.collection(collName);
 
-  if(!newObj.title){
+  if (!newObj.title) {
     // Invalid movie object, shouldn't go in database.
-    return {error: "Movies must have a title."}
+    return { error: "Movies must have a title." }
   }
   const result = await movies.insertOne(newObj);
 
-  if(result.acknowledged){
+  if (result.acknowledged) {
     return { newObjectId: result.insertedId, message: `Item created! ID: ${result.insertedId}` }
   } else {
-    return {error: "Something went wrong. Please try again."}
+    return { error: "Something went wrong. Please try again." }
   }
 }
 
-module.exports.createComment = async(movieId, newObj) =>{
+module.exports.createComment = async (movieId, newObj) => {
   // TODO: Validate that movieId is for an existing movie
   const database = client.db(databaseName);
   const comments = database.collection(commCollName);
-
-  const goodObj = {...newObj, movie_id: ObjectId(movieId), date: new Date()}
+  const validatedMovie = await module.exports.getById(movieId);
```

Comment:
This is a good addition, but the validation would be most useful if it was checked before the new comment was created on line 108, rather than just before sending a response.